src/Components/FeedbackIcon/feedbackIcon.css

```diff
@@ -0,0 +1,67 @@
+.feedback-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  width: 18.75rem;
```

Comment:
The width needs to be adjusted automatically because different languages could take up different amounts of space.